modules/enterprise/agent/src/main/java/org/rhq/enterprise/agent/AgentConfigurationConstants.java

```diff
@@ -162,6 +162,16 @@
     String AGENT_UPDATE_DOWNLOAD_URL = PROPERTY_NAME_PREFIX + "agent-update.download-url";
 
     /**
+     * The amount of milliseconds the agent will wait before force closing after an update.
+     */
+    String AGENT_UPDATE_EXIT_TIMEOUT_MSECS = PROPERTY_NAME_PREFIX + "agent-update-exit-timeout-msecs";
+
```

Comment:
Not sure if I should change the name to: "AGENT_EXIT_TIMEOUT_MSECS", to cover as a global exit timeout (not sure if is or will be used anywhere else).